lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -1074,4 +1075,31 @@ sub sdaf_register_byos {
     );
 }
 
+=head2 sdaf_deployment_reused
+
+    sdaf_deployment_reused(quiet=>'BeQuiet!');
+
+If an existing deployment is being reused according to openQA setting `SDAF_DEPLOYMENT_ID`, function will display
+`record_info` message with details and returns deployment ID. Otherwise returns nothing/false.
+Argument B<quiet> can be used to disable `record_info` message.
+
+=over
+
+=item * B<quiet>: Hide 'record_info' message. Default: undef
+
+=back
+=cut
+
+sub sdaf_deployment_reused {
+    my (%args) = @_;
+    my $deployment_id = get_var('SDAF_DEPLOYMENT_ID');
```

Comment:
From where SDAF_DEPLOYMENT_ID is coming from? Are we sure it couldn't  be  some forgotten value? If not, would be fine to also make some simple check that deployment really exist.

Reply:
SDAF_DEPLOYMENT_ID is defined only as test setting on cmdline in case they want to reuse existing deployment. 
Chcecks are done in other places. There is for example whole test module just for that: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/redirection_tests/redirection_check.pm

This function serves just as s flag that no deployment code should be executed.

Reply:
TY

Reply:
TY